client/src/pages/course/mentor/confirm.tsx

```diff
@@ -31,21 +33,22 @@ function Page(props: { session: Session; courseAlias?: string }) {
   useAsync(async () => {
     try {
       setLoading(true);
-      const courseAlias = router.query?.course;
-      if (!courseAlias) {
+      const courseAlias = router.query?.course as string;
+      if (courseAlias == null) {
         return;
       }
       const courses = await new CoursesService().getCourses();
-      const course = courses.find(c => c.alias === courseAlias) ?? null;
+      const course = courses.find(c => c.alias.toLowerCase() === courseAlias.toLowerCase()) ?? null;
       setCourse(course);
       const mentor = await mentorRegistry.getMentor();
+      const preferredCourse = mentor.preferredCourses?.includes(course?.id ?? 0);
```

Comment:
What does it mean `0`?

Reply:
This is the old logic, I didn't change it. I guess the course with id = 0 does not exist, so in this way we exclude any coincidence of courses